core/unix/src/TUnixSystem.cxx

```diff
@@ -2857,8 +2857,8 @@ const char *TUnixSystem::GetLinkedLibraries()
       char* longerexe = new char[lenexe + 5];
       strlcpy(longerexe, exe,lenexe+5);
       strlcat(longerexe, ".exe",lenexe+5);
-      delete [] exe;
-      exe = longerexe;
+      // delete [] exe; // dead code, but operation is not allowed
```

Comment:
How did you notice it was an operation that is not allowed?

Reply:
>  it was an operation that is not allowed?

I guess, it can be compiled. But of course, it is failure, while ones tries to delete memory which is not owned at this place.